lib/linux_admin/network_interface.rb

```diff
@@ -16,6 +16,8 @@ def self.dist_class(clear_cache = false)
       @dist_class ||= begin
         if [Distros.rhel, Distros.fedora].include?(Distros.local)
           NetworkInterfaceRH
+        elsif Distros.mac == Distros.local
+          NetworkInterfaceMac
```

Comment:
Maybe this can all be:
```ruby
case(Distros.local)
when Distros.rhel, Distros.fedora
  NetworkInterfaceRH
when Distros.mac
  NetworkInterfaceMac
else
  NetworkInterfaceGeneric
end
```
I think it's nicer than the array include followed by `==`